src/main/java/com/gregtechceu/gtceu/data/pack/GTDynamicResourcePack.java

```diff
@@ -138,7 +144,7 @@ public static void writeByteArray(ResourceLocation id, @Nullable String subdir,
     @Nullable
     @Override
     public InputStream getRootResource(String elements) {
-        throw new UnsupportedOperationException("Dynamic Resource Pack cannot have root resources");
+        return this.getClass().getResourceAsStream(elements);
```

Comment:
Is this intentionally only in the cherrypick, or should it be in 1.20 too?

Reply:
1.20 does it differently iirc?

Reply:
lemme test

Reply:
should just return null here probably...

Reply:
Hmm, makes sense I think ^^